pyproject.toml

```diff
@@ -26,7 +26,6 @@ classifiers = [
     "Programming Language :: Python :: 3.11",
```

Comment:
Optional: you may also add support for Python 3.13. Here is a quick list of places to change the version: https://github.com/search?q=repo%3Abryzgaloff%2Fairflow-clickhouse-plugin%203.12&type=code

Make sure to exclude Python-Airflow combinations in the `.github/workflows/tests.yml` file.

Not required, but if you wish, you may add the change to this PR or create a new one. You will be mentioned in the contributors list either way :)

Reply:
Ok, sure, I will update myself later 🤝